mlir_graphblas/engine.py

```diff
@@ -724,7 +704,7 @@ def _generate_multivalued_functions(
             def python_callable(*args) -> tuple:
                 if len(args) != len(mlir_function.args):
                     raise ValueError(
-                        f"{name} expected {len(mlir_function.args)} args but got {len(args)}."
+                        f"{mlir_function.name} expected {len(mlir_function.args)} args but got {len(args)}."
```

Comment:
I think that `mlir_function.name` should actually be `mlir_function.name.value` since
```
>>> import mlir
>>> mlir_ast = mlir.parse_string('''
func @function_2() {
  return
}
''')
>>> mlir_ast.body[0]
Function(name=SymbolRefId(value=function_2), args=[], result_types=[], attributes=None, body=Region(body=[Block(label=None, body=[Operation(result_list=[], op=ReturnOperation(values=None, types=None), location=None)])]), location=None, visibility=public)
>>> mlir_ast.body[0].name
SymbolRefId(value=function_2)
>>> mlir_ast.body[0].name.value
'function_2'
>>> 
```
This is a bug I shouldn't've let through when I originally added this code.